docs/sql-ref-syntax-qry-select.md

```diff
@@ -18,8 +18,132 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+Spark supports `SELECT` statement and conforms to ANSI SQL standard. Queries are
+used to retrieve result sets from one or more table. The following section 
+describes the overall query syntax and the sub-sections cover different constructs
+of a query along with examples. 
 
-Spark SQL is a Apache Spark's module for working with structured data.
-This guide is a reference for Structured Query Language (SQL) for Apache 
-Spark. This document describes the SQL constructs supported by Spark in detail
-along with usage examples when applicable.
+### Syntax
+{% highlight sql %}
+[WITH with_query [, ...]]
+SELECT [hints, ...] [ALL|DISTINCT] named_expression[, named_expression, ...]
+  FROM from_item [, from_item, ...]
```

Comment:
lateral views and pivot is out-of-scope here? https://github.com/apache/spark/blob/master/sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4#L569